mon-pix/tests/unit/components/routes/campaigns/skill-review_test.js

```diff
@@ -260,7 +260,7 @@ describe('Unit | component | Campaigns | Evaluation | Skill Review', function()
         it('should add the masteryPercentage to the url', function() {
```

Comment:
**suggestion** : changer le titre des `it` + communiquer que ça dans l'url récupéré par les clients, ce n'est plus un % mais un taux

Reply:
Non j'ai pas changé l'URL on envoie toujours un mastery % dans l'URL